package.json

```diff
@@ -119,7 +119,7 @@
     "@pmmmwh/react-refresh-webpack-plugin": "0.5.0-beta.8",
     "@schemastore/web-manifest": "0.0.5",
     "@stacks/prettier-config": "0.0.8",
-    "@stacks/stacks-blockchain-api-types": "0.62.1",
+    "@stacks/stacks-blockchain-api-types": "blockstack/stacks-blockchain-api#feat/microblocks",
```

Comment:
@beguene new version was published to use:
https://www.npmjs.com/package/@stacks/stacks-blockchain-api-types/v/0.63.0-beta.0

Reply:
Thanks !